### DISCUSSION TITLE
Gap between operations at the start of the previous one, not at the end.

### DISCUSSION THREAD

**Discussion by salvastp** [frePPLe/frepple#615] [General]
Hello everyone,

After researching, I can't seem to find a way to sequence two operations with the following characteristics:

- They are operations on the same item, but on two different machines (cutting and drilling).
- Both operations are dependent and occur at the same rate.
- The second operation can start after a certain amount of time has passed since the start of the first one (let's say it has a safety buffer).
- Its like a "safety_leadtime" of the operations dependency, but at he start, not a the end.

What would be the most correct way to represent this situation?

Thank you.

**Comment by jdetaeye** [comment]
Option1;
See https://frepple.com/docs/current/user-guide/examples/buffer/transfer-batch.php for an example of a feature called "transfer batching".  It was designed to model situations where items are moved between production steps in smaller quantities/batches (eg pallets, boxes, ...) rather than when completely finished.
Warning:  The feature complicates the planning significantly. We have also seen that it makes the plan more complex and unintuitive to interactively change.    My recommendation is to stay away from this feature if you can...

Option 2:
Alternatively you can explore operation dependencies https://frepple.com/docs/current/user-guide/examples/operation/operation-dependency.php to model dependencies.  This is a more convenient way of modeling overlapping operations, but the dependencies are based on the end time.   No way (yet) to define them based on the start date.

**Comment by salvastp** [comment]
Thank you for your response,

I've tried both options.

The first one overly complicates the model, and it doesn't properly align with the problem.

I've experimented with operation dependencies (even using negative dependency times), but as you mentioned, the dependencies currently rely on the end of the operation, not the beginning.

This isn't a problem for short-duration operations (hours), but if they are longer (several days), the planning result stretches over a long period, making it less representative.

It would be a good option to be able to define this time based on the start date.

Thank you for your time.
